Documentation/ApiOverview/Events/Events/Core/Package/AfterPackageActivationEvent.rst

```diff
@@ -6,9 +6,36 @@
 AfterPackageActivationEvent
 ===========================
 
-The PSR-14 event :php:`\TYPO3\CMS\Core\Mail\Event\AfterPackageActivationEvent`
+..  versionadded:: 10.3
+    The event was introduced to replace the Signal/Slot
+    `\TYPO3\CMS\Extensionmanager\Utility\InstallUtility::afterExtensionInstall`.
+
+The PSR-14 event :php:`\TYPO3\CMS\Core\Package\Event\AfterPackageActivationEvent`
 is triggered after a package has been activated.
 
+..  attention::
+    This event is only dispatched when an extension is activated in the
+    :guilabel:`Extension Manager`, therefore starting with TYPO3 v11 this
+    event is only dispatched in legacy installations, not in Composer-based
```

Comment:
"event is only dispatched" used twice in the sentence. Maybe write something like

"This event is <del>only</del> dispatched when an extension is activated in the Extension Manager, therefore starting with TYPO3 v11 it is only fired in legacy installations, not in Composer-based ..."

Also for the other events.